src/ngraph/codegen/file_util.hpp

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+* Copyright 2017-2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <functional>
+#include <string>
+#include <vector>
+
+namespace ngraph
+{
+    namespace codegen
+    {
+        class file_util;
+    }
+}
+
+class ngraph::codegen::file_util
```

Comment:
i'm a bit confused why this is a class and not a namespace, but :okay:

Reply:
I agree -- it should be a namespace.